types/lib/components/Feedback.d.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare global {
+  export namespace WebFeedback {
```

Comment:
Should we be namespacing these typings with `Web` or would it make sense to instead call them `Feedback`?

Reply:
I chose this because `Web` is the prefix of our custom elements, I'm not sure how having the namespace of `Web` declared in different files would work though...